File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.BudgetsService.Models/budget-item_item.ts



export class BudgetItem_Item {
  
	public BudgetItemID?: number;
	public TrainingEventID: number = 0;
	public IsIncluded: boolean = false;
	public LocationID?: number;
	public BudgetItemTypeID: number = 0;
	public BudgetItemType: string = "";
	public BudgetCategoryID: number = 0;
	public BudgetCategory: string = "";
	public Cost: number = 0;
	public Quantity?: number;
	public PeopleCount?: number;
	public TimePeriodsCount?: number;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}

File: GTTS/INL.BudgetsService/Models/BudgetItem_Item.cs
using System;

namespace INL.BudgetsService.Models
{
    public class BudgetItem_Item
    {
        public long? BudgetItemID { get; set; }
        public long TrainingEventID { get; set; }
        public bool IsIncluded { get; set; }
        public long? LocationID { get; set; }
        public int BudgetItemTypeID { get; set; }
        public string BudgetItemType { get; set; }
        public int BudgetCategoryID { get; set; }
        public string BudgetCategory { get; set; }
        public decimal Cost { get; set; }
        public int? Quantity { get; set; }
        public int? PeopleCount { get; set; }
        public decimal? TimePeriodsCount { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}
